src/modules/tests/instructor-details/instructor-details.actions.ts

```diff
@@ -4,7 +4,7 @@ export const INSTRUCTOR_REGISTRATION_NUMBER_CHANGED = '[Vehicle Details] Instruc
 
 export class InstructorRegistrationNumberChanged implements Action {
   readonly type = INSTRUCTOR_REGISTRATION_NUMBER_CHANGED;
-  constructor(public instructorRegistrationNumber: string) {}
+  constructor(public instructorRegistrationNumber: number) {}
```

Comment:
Can we confirm with POs that this number will never have a leading 0?

Reply:
have requested further information from POs. will revisit if their answer indicates we need to treat registration number as a string